<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_Interface</class>
 <widget class="QMainWindow" name="User_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User_Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tablero">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>681</width>
      <height>311</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QPushButton" name="Activa_Man_Enc">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Encender</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Activa_Man_Apa">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Apagar</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Conf_Temp_Man">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>18</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cambio de Temperatura</string>
      </property>
     </widget>
     <widget class="Led" name="Estado_Aire">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Color Led component</string>
      </property>
      <property name="whatsThis">
       <string>Led indicator</string>
      </property>
      <property name="color">
       <color>
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Estado Dispositivo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Encendido del Dispositivo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Apagado del Dispositivo</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Programable</string>
     </attribute>
     <widget class="QPushButton" name="Activa_Prog_Enc">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Encender Programado</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Activa_Prog_Apa">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Apagar Programado</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hora de Encendido</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hora de Apagado</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="Hora_Prog_Enc">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="time">
       <time>
        <hour>12</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
     <widget class="QTimeEdit" name="Hora_Prog_Apa">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::MinuteSection</enum>
      </property>
      <property name="displayFormat">
       <string>hh:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="currentSectionIndex">
       <number>1</number>
      </property>
      <property name="time">
       <time>
        <hour>12</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
     <widget class="QDateEdit" name="Fecha_Prog_Enc">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>12</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="time">
       <time>
        <hour>12</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha de Encendido</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="Fecha_Prog_Apa">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>12</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="time">
       <time>
        <hour>12</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha de Apagado</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperatura de Encendido</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Temp_Prog_Enc">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>18</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Estado Programado Encendido</string>
      </property>
     </widget>
     <widget class="Led" name="Led_Prog_Enc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Color Led component</string>
      </property>
      <property name="whatsThis">
       <string>Led indicator</string>
      </property>
      <property name="color">
       <color>
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </property>
     </widget>
     <widget class="Led" name="Led_Prog_Apa">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>160</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Color Led component</string>
      </property>
      <property name="whatsThis">
       <string>Led indicator</string>
      </property>
      <property name="color">
       <color>
        <red>0</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Estado Programado Apagado</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Cancela_Programado">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancelar Programado</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="List_Disp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>Mundo Clima</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Panasonic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fujitsu</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sistema Conectado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Activa_Conex">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Validar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Topic de Conexión</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Topic_Conex">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>/TFG/ConectarAA</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>691</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="Estado_Button">
     <property name="text">
      <string>Estado:</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Detenido</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Broker</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Text_Broker">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>broker.hivemq.com</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>analogwidgets/led.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
